### PR TITLE
Add ARM64 Linux to CI matrix

### PR DESCRIPTION
**Pull request by MarkMcCaskey**
# Review

- [ ] Add a short description of the the change to the CHANGELOG.md file

### CODE REVIEW

**Comment by MarkMcCaskey** [comment]
bors try

**Comment by MarkMcCaskey** [comment]
bors try-

**Comment by MarkMcCaskey** [comment]
bors try

**Comment by MarkMcCaskey** [comment]
bors try

**Comment by MarkMcCaskey** [comment]
Interesting failure https://github.com/wasmerio/wasmer/runs/1103249251

looks like we need libclang to build libffi for the  CAPI to work on ARM

**Comment by MarkMcCaskey** [comment]
bors try

**Comment by MarkMcCaskey** [comment]
bors try

**Comment by MarkMcCaskey** [comment]
bors try-

**Comment by MarkMcCaskey** [comment]
bors try

**Comment by MarkMcCaskey** [comment]
bors try-

**Comment by MarkMcCaskey** [comment]
bors try

**Comment by MarkMcCaskey** [comment]
bors try-

**Comment by MarkMcCaskey** [comment]
bors try

**Comment by MarkMcCaskey** [comment]
bors try

**Comment by MarkMcCaskey** [comment]
bors try

**Comment by MarkMcCaskey** [comment]
bors try-

**Comment by MarkMcCaskey** [comment]
bors try

File: .github/workflows/main.yaml [hunk 1]
```diff
@@ -198,9 +200,19 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         with:
           upload_url: ${{ steps.create_release.outputs.upload_url }}
-          asset_path: artifacts/wasmer-macos-latest/wasmer-darwin-amd64.tar.gz
+          asset_path: artifacts/wasmer-macos-amd64/wasmer-darwin-amd64.tar.gz
           asset_name: wasmer-darwin-amd64.tar.gz
           asset_content_type: application/gzip
+      - name: Upload Release Asset Linux aarch64
```
Comment (nlewycky) [inline .github/workflows/main.yaml:206]: I'd like to be consistent about "arm" vs. "aarch64". I'm fine with either myself (I live my best life by pretending arm32 doesn't exist) but if you want to leave open the possibility of 32-bit arm in the future, change 'arm' to 'aarch64' throughout this PR.

Comment (MarkMcCaskey) [reply .github/workflows/main.yaml:206]: Done! All the remaining uses are required as that's the label that Github Actions uses (`ARM64`)

**Review by nlewycky — APPROVED** [review]

File: bors.toml [hunk 2]
```diff
@@ -2,6 +2,7 @@ status = [
     "Audit",
     "Code lint",
     "Test on linux",
+    "Test on linux-arm",
```
Comment (nlewycky) [inline bors.toml:5]:
```suggestion
    "Test on linux-aarch64",
```

**Pull request merged**